tt_metal/jit_build/hlk_desc.hpp

```diff
@@ -34,6 +34,13 @@ class tt_hlk_desc
     DataFormat param_buf_dataformat_arr[8];
     DataFormat output_buf_dataformat_arr[8];
     DataFormat intermediate_buf_dataformat_arr[8];
+    uint32_t buf_num_faces_arr[32];
```

Comment:
Replace 32 with NUM_CIRCULAR_BUFFERS or something ?

Reply:
don't think this file has that included.
please note this is following a similar style as the current code

Reply:
Ok I marked as resolved the other comments regarding 32 magic number